docs/contributing/docs-templates.md

```diff
@@ -3,69 +3,82 @@ title: Docs Templates
 
-### Near-perfect example of a guide article
+Reference guide sections provide canonical information on how and why to build things with Gatsby for a variety of scenarios.
 
-[Add a 404 page](/docs/add-404-page/) is an example of a guide that isn't perfect yet. As you read through this template, take note of where that article can be improved.
+### What should a reference guide be about?
 
-### What should a guide article be about?
+We need guide articles to describe every concept and task you can accomplish with Gatsby.
 
-We need guide articles to describe every task you can accomplish with Gatsby.
+Each guide article should explain exactly one concept and that concept should be apparent from the article's title. [Overview guides](#reference-guide-overview) can list child pages to present multiple ways of getting a job done while limiting the scope of each individual article (e.g. Styling Your Site, Using Layout Components, Standard Global CSS Files, etc.)
 
-Each guide article should explain exactly one task and that task should be
-apparent from the article's title.
+### Near-perfect example of a reference guide
 
-#### What if I want to include multiple tasks and concepts in a guide article?
+[Linking Between Pages](/docs/linking-between-pages/)
 
-If you find yourself wanting to include multiple related topics in one article, consider splitting each into its own individual guide and referencing the other topics under sections called “Prerequisites” and/or "Other
-Resources" sections in the related guide articles.
+### What if I want to include multiple tasks and concepts in a reference guide?
+
+If you find yourself wanting to include multiple related topics in one article, consider splitting each into its own individual guide and referencing the other topics under sections called “Prerequisites” and/or "Other Resources" sections in the related guide articles.
 
 It’s more ideal to have many articles that cover a broad range of technical topics rather than smashing too many topics into one article.
 
-If you find yourself wanting to teach the reader how to accomplish a series of related tasks, you might want to write a tutorial.
+If you find yourself wanting to teach the reader how to accomplish a series of related tasks, you might want to write a tutorial. For short and super common how-to instructions for a single task, a recipe may work best.
 
-#### When to write a guide vs. a tutorial?
+### When to write a reference guide vs. a tutorial, vs. a recipe?
 
-A guide explains a discrete task without the step-by-step context provided by a tutorial.
+[Reference guide articles](#reference-guide-template) cover discrete topics as documentation while linking to other resources and guides. A reference guide explains a task or concept without the step-by-step context provided by a tutorial or recipe.
```

Comment:
```suggestion
[Reference guide articles](#reference-guide-template) cover discrete topics such as documentation while linking to other resources and guides. A reference guide explains a task or concept without the step-by-step context provided by a tutorial or recipe.
```

I think this is what you meant? ^

Reply:
Nope, it was written that way to say the topics are documentation. If it's confusing we can change it, though.

Reply:
Ah right. Makes sense 👍